Index: public/config.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/public/config.php b/public/config.php
new file mode 100644
--- /dev/null	(date 1727810213523)
+++ b/public/config.php	(date 1727810213523)
@@ -0,0 +1,29 @@
+<?php
+
+class config
+{
+
+    private $host = '127.0.0.1';
+    private $db = 'user_data';
+    private $user = 'postgres';
+    private $pass = '';
+    private $port = '5432';
+    private $charset = 'utf8';
+    private $pdo;
+
+    public function __construct()
+    {
+        $dsn = "pgsql:host={$this->host};port={$this->port};dbname={$this->db};user={$this->user};password={$this->pass}";
+        $options = [
+            PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION,
+            PDO::ATTR_DEFAULT_FETCH_MODE => PDO::FETCH_ASSOC,
+            PDO::ATTR_EMULATE_PREPARES => false,
+        ];
+
+        try {
+            $this->pdo = new PDO($dsn, $options);
+        } catch (\PDOException $e) {
+            throw new \PDOException($e->getMessage(), (int)$e->getCode());
+        }
+    }
+}
